src/gui/gui.cpp

```diff
@@ -23,22 +23,20 @@
 #include "gui/core/log.hpp"
 #include "gui/core/gui_definition.hpp"
 #include "gui/widgets/settings.hpp"
-#include "preferences/preferences.hpp"
 #include "serialization/parser.hpp"
 #include "serialization/preprocessor.hpp"
 #include "serialization/schema_validator.hpp"
 #include "wml_exception.hpp"
 
 namespace gui2
 {
-static bool initialized = false;
+
+namespace {
+config guis_cfg;
```

Comment:
This doesn't seem to be necessary anymore.

Reply:
NVM, removed.